api/main.py

```diff
@@ -92,9 +92,9 @@ def root():
                 for parallel in segment["parallels"]:
                     parallel_count += 1
                     for seg_nr in parallel:
-                        collection_key = re.search(r"^[A-Z]+[0-9]+", seg_nr).group()
-                        if collection_key not in collection_keys:
-                            collection_keys.append(collection_key)
+                        collection_key = re.search(r"^[a-zA-Z]+[0-9]+", seg_nr)
```

Comment:
I changed my mind on this one. The collection keys for the pali are not `[a-zA-Z]+[0-9]+` but `[a-z]+` only. F.I. the collection is `dn` with filenames `dn1, dn2, etc`. This is different from the Tibetan and other languages.

Reply:
OK, good to know, thanks! You can request changes in the code review.